docfx_project/examples/Playwright.Logger.md

```diff
@@ -11,7 +11,7 @@ Playwright Sharp uses a [.NET Logger Factory](https://docs.microsoft.com/en-us/d
 
 It uses two categories:
  * All the communication between the library and the driver is under the `PlaywrightSharp.Transport.Connection` category.
- * The debug information coming from the driver, when `debug: true` is set, is under the `PlaywrightSharp.Playwright` category.
+ * The debug information coming from the driver, when `debug: "pw:api"` is set, is under the `PlaywrightSharp.Playwright` category.
```

Comment:
Consider noting here that the value is passed to the npm debug module (to the nodejs process that passes it to the debug module)